ts/core/MmlTree/MmlNode.ts

```diff
@@ -303,6 +315,7 @@ export abstract class AbstractMmlNode extends AbstractNode<MmlNode, MmlNodeClass
   public static verifyDefaults: PropertyList = {
     checkArity: true,
     checkAttributes: false,
+    checkMathvariants: true,
```

Comment:
Why do we need an extra option here? Could that not just be subsumed by `checkAttributes`?